### DISCUSSION TITLE
OTB100的结果为什么只有没fineturn附件的结果0.673？

### DISCUSSION THREAD

**Discussion by ghost** [STVIR/pysot#275] [General]
原码        dist_model = DistModule(model)在model = ModelBuilder().cuda().train()后面，现在为什么改为10几句后面才出现呢？OTB100的结果为什么只有没fineturn附件的结果0.673？我fineturn了的。

**Comment by FrankYoungchen** [comment]
可以咨询一下OTB100如何下载嘛？

**Comment by ZhiyuanChen** [comment]
很抱歉，我无法理解你的问题。
什么叫做OTB100的结果为什么只有没fineturn附件的结果？
